contrib/update-lock-files.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+#
+# Update the minimal/recent lock file
+
+set -euo pipefail
+
+for file in Cargo-minimal.lock Cargo-recent.lock; do
+    cp -f "$file" Cargo.lock
+    cargo check
```

Comment:
Does cargo check actually update the lock file? Shouldn't it be `cargo update`?

Reply:
I checked that this indeed does update the lock files by checking out #373, checking out the `Cargo-recent.lock` and `Cargo-minimal.lock` files from it's parent a9b9a34183b8776625f6ab6a752eab5267407737, and running the script. Indeed it does actually update the lock file. I made an edit to OP sharing that this script came directly from rust-bitcoin.